content/posts/2020/05-03-Tutorial-Combining-Nornir-&-pyATS.md

```diff
@@ -0,0 +1,146 @@
+---
+title: "Tutorial: Combining Nornir with pyATS"
+date: "2020-05-03"
+tags: ["nornir", "pyATS"]
+---
+
+This tutorial aims to provide a simple use-case for combining Nornir and pyATS together in order to profile your current network configurations and implement desired state - as specified in host variable definition files.
+
+The workflow we will follow in this tutorial will be first:
+- Deploy OSPF desired state using Nornir
+- Immediately deploying pyATS to capture a golden image profile of this &quot;desired state&quot;
+----------------------------------------------------------------------------------------------------
+
+After we have conducted the initial steps we will then deploy a python script (Pynir.py) that will:
```

Comment:
it would be nice to format code, variable names, filenames, etc. consistently with backticks. Sometimes there is nothing and sometimes you use italic, preformatted using backticks might look nicer